proxy/proxy.go

```diff
@@ -973,6 +980,9 @@ func (p *Proxy) getRoundTripper(ctx *context, req *http.Request) (http.RoundTrip
 		if sf, ok := ctx.StateBag()["fastCgiFilename"]; ok {
 			f = sf.(string)
 		}
+		if req.URL.Path[1:] != "" {
```

Comment:
Might be a case for an else condition here as the explicit filename would take preference

Reply:
I think also statebag entry should have precedence, here the statebag decision would be overridden.

Reply:
```go
else if req.URL.Path != "/" {
```

Reply:
thanks ! done